.github/workflows/ci-owasp.yaml

```diff
@@ -0,0 +1,94 @@
+name: owasp-scan
+on:
+  workflow_call:
+  pull_request:
+    inputs:
+      ZAP_SCAN_TYPE:
+        required: true
+        type: string
+      ZAP_TARGET_URL:
+        required: true
+        type: string
+      ZAP_DURATION:
+        required: true
+        type: string
+      ZAP_MAX_DURATION:
+        required: true
+        type: string
+      ZAP_GCP_PUBLISH:
+        required: false
+        type: boolean
+      ZAP_GCP_PROJECT:
+        required: false
+        type: string
+      ZAP_GCP_BUCKET:
+        required: false
+        type: string
+    secrets:
+      GCP_SA_KEY:
+        required: false # only needed if Publish is true
+jobs:
+  owasp-scan:
+    runs-on: ubuntu-latest
+    name: owasp-scan
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

Comment:
I think we will need to use v3 now. I updated the other dev/test actions but I am wondering now if the "promote prod" GHA needs to be updated as well? (b/c it is failing...?)